tests/test_interface.rb

```diff
@@ -1350,20 +1325,18 @@ def test_interface_ipv4_all_interfaces
     rescue Minitest::Assertion
       # clean up before failing
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
       raise
     end
   end
 
   def test_interface_vrf_default
-    config('interface loopback1', 'vrf member foo')
     interface = Interface.new('loopback1')
     interface.vrf = interface.default_vrf
     assert_equal(DEFAULT_IF_VRF, interface.vrf)
   end
 
   def test_interface_vrf_empty
-    config('interface loopback1', 'vrf member foo')
```

Comment:
I think combining the two tests into one should resolve this and remove the redundancy of the tests:

1. Create loopback interface
1. Assert vrf is empty
1. Set vrf to `foo`
1. Assert vrf value is `foo`
1. Set vrf to default value
1. Assert vrf is default value
1. Cleanup interface